web-registry/src/lib/metadata-graph.ts

```diff
@@ -1,12 +1,14 @@
-import axios, { AxiosResponse } from 'axios';
+import axios from 'axios';
 import getApiUri from './apiUri';
 
-export const getMetadata = async (hash: Uint8Array): Promise<AxiosResponse> => {
+export const getMetadata = async (hash: Uint8Array): Promise<any> => {
   if (hash) {
     // The on-chain metadata fields being stored as bytes for now, we need to decode it using base64 to get the corresponding IRI value as a string.
     // This won't be needed once the on-chain metadata fields are converted to string: https://github.com/regen-network/regen-ledger/issues/829
     const iri = atob(hash.toString());
-    return axios.get(`${getApiUri()}/metadata-graph/${iri}`);
+    const { data } = await axios.get(`${getApiUri()}/metadata-graph/${iri}`);
+
+    return data;
```

Comment:
i decided to actually await the response here, and get the data. this means this function really returns the metadata, so it makes our call sites a little cleaner i think.